winui/AutoComplete/Highlighting.md

```diff
@@ -0,0 +1,187 @@
+                            DisplayMemberPath="Name"                       
+                            TextHighlightMode="Matched"                                
+                            TextSearchMode="StartsWith"                               
+                            HighlightedTextFontSize="15"                               
+                            HighlightedTextFontStyle="Italic"                               
+                            HighlightedTextFontWeight="Medium"                                
+                            HighlightedTextForeground="Red"                                
+                            ItemsSource="{Binding SocialMedias}">       
+    </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% highlight C# %}
+        autoComplete.DisplayMemberPath = "Name";
+        autoComplete.TextMemberPath = "Name";
+        autoComplete.TextHighlightMode = AutoCompleteTextHighlightMode.Unmatched;
+        autoComplete.TextSearchMode = AutoCompleteTextSearchMode.Contains;
+        autoComplete.HighlightedTextFontSize = 10;
+        autoComplete.HighlightedTextFontStyle = FontStyle.Normal;
+        autoComplete.HighlightedTextFontWeight = FontWeights.ExtraLight;
+        autoComplete.HighlightedTextForeground = Color.Red;
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete text highlighting properties](Highlighting_images/winui-autocomplete-highlightingproperties.png)
+
+
+
+### **Highlight beginning text**
+It highlights the matches that starts with the typed characters in suggestion list.
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfAutoComplete x:Name="autoComplete"                            
+                        DisplayMemberPath="Name"                             
+                        TextHighlightMode="Matched"
+                        TextSearchMode="StartsWith"                           
+                        HighlightedTextForeground="Red"                               
+                        ItemsSource="{Binding Countries}">
+</editors:SfAutoComplete>
+{% endhighlight %}
+{% highlight C# %}
+
+autoComplete.DisplayMemberPath = "Name";
+autoComplete.TextHighlightMode = AutoCompleteTextHighlightMode.Matched;
+autoComplete.TextSearchMode = AutoCompleteTextSearchMode.StartsWith;
+autoComplete.HighlightedTextForeground = Color.Red;
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+
+![WinUI AutoComplete text highlighting based on begining text one character](Highlighting_images/winui-autocomplete-textsearchmode-startswith-onecharacter.png)
+
+### Searching with more than one character
+
+![WinUI AutoComplete text highlighting based on begining text more than one character](Highlighting_images/winui-autocomplete-textsearchmode-startswith-morethanonecharacter.png)
+
+
+
+
+### **Highlight all occurance of search text**
+It highlights all the matches that contains the typed characters in suggestion list. 
```

Comment:
It highlights all the matches that contain the typed characters in the suggestion list.